package.json

```diff
@@ -15,7 +15,7 @@
   "license": "BSD-3-Clause",
   "scripts": {
     "lint": "eslint .",
-    "test-node": "mocha test/ integration-test/ -R dot --check-leaks",
+    "test-node": "mocha --timeout 200 test/ integration-test/ -R dot --check-leaks",
```

Comment:
That's a pretty big smell from replacing `var` with `const` and `let`.

Is this really necessary?

Reply:
Not sure I agree: that was just an improvement of our general setup that I stumbled upon when doing this. Without it, if you should remove the `done` callback in any of our tests it would just hang for 2 whole seconds, which was a PITA when doing this, as so many tests were breaking. None of our tests should use anywhere near 100 ms, but I was just going conservative when setting it to 200 ms. 

Before:
```
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/carlerik/dev/fake-timers/test/fake-timers-test.js)
```

After:
```
     Error: Timeout of 200ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/carlerik/dev/fake-timers/test/fake-timers-test.js)
```

We could of course stuff it in a .mocharc or something, but we do not really have _that_ many options to hide. What do you suggest?

Reply:
When I read it, I thought that we would accept tests running for `200ms` as acceptable. Which seems excessive.

I guess that's what we're actually doing though.

Thank you for the explanation 👍